packages/api/orm/entity/Clin.ts

```diff
@@ -0,0 +1,51 @@
+import { BaseEntity } from "./BaseEntity";
+import { Column, Entity, ManyToOne } from "typeorm";
+import { TaskOrder } from "./TaskOrder";
+
+@Entity()
+export class Clin extends BaseEntity {
+  @Column({
+    type: String,
+    nullable: false,
+    length: 4,
+    comment: "contract line item number from task order, 0001 through 9999",
```

Comment:
Will updating these comments later require a database schema update? Do we want to commit to that potential maintenance burden for nearly every column if so?

Reply:
No, I don't believe it would require a schema update.  The table and column comments can be updated discretely with the [comment command](https://www.postgresql.org/docs/13/sql-comment.html).  They can be viewed in Postgres within `psql` using the `\d+ <tablename>` command.

I think it's worth potential maintenance burden for fields containing important business information that may not be clear from field names.  I purposefully didn't place them everywhere, because some things are self evident.  I imagine that if I was maintaining (or querying) this system far in the future that it might be helpful to have descriptions for fields immediately next to the data, at least for entities such as `clin`.